sdk/purview/Azure.Analytics.Purview.Catalog/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-09-30)
```

Comment:
```suggestion
## 1.0.0-beta.2 (2021-10-05)
```

We typically release package the non core package on the Tuesday after code complete, so I updated the date here.